examples/cms/src/App.tsx

```diff
@@ -0,0 +1,134 @@
+import { framer, ManagedCollectionField } from "framer-plugin"
+import "./App.css"
+import { useLayoutEffect, useState } from "react"
+import { DataSource, getDataSources, syncCollection, syncExistingCollection } from "./data"
+import { FieldMapping } from "./FieldMapping"
+import { PLUGIN_KEYS, UI_DEFAULTS } from "./constants"
+import { Spinner } from "./components/Spinner"
+
+const activeCollection = await framer.getManagedCollection()
+
+const syncDataSourceId = await activeCollection.getPluginData(PLUGIN_KEYS.SYNC_REFERENCE)
+const syncSlugFieldId = await activeCollection.getPluginData(PLUGIN_KEYS.SYNC_SLUG)
```

Comment:
What does the prefix _sync_ mean here?

Reply:
As in `configuredDataSourceId`, sync means that it has been imported.

Reply:
That's a much better name. The term _sync_ can mean too many things